src/main/java/org/mastodon/util/MastodonDebugSettings.java

```diff
@@ -143,9 +144,9 @@ private static Yaml createYaml()
 		{
 			final DumperOptions dumperOptions = new DumperOptions();
 			dumperOptions.setDefaultFlowStyle( DumperOptions.FlowStyle.BLOCK );
-			final Representer representer = new Representer();
+			final Representer representer = new Representer( new DumperOptions() );
```

Comment:
There is a dumperOptions constructed 2 lines above. Why not use that instead of a new one?

Reply:
@tpietzsch thanks for your comments. The suggestions are good. I closed this PR because of https://github.com/mastodon-sc/mastodon/pull/261#issuecomment-1761275957